ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-arm.yml

```diff
@@ -97,6 +107,9 @@
     hand_side_label: "Left Arm and Hand"
     control_loop_script_name: "shadow_nuc_left_arm_hand_hardware_control_loop.sh"
     gui_script_name: "shadow_GUI_left_arm_hand.sh"
+    enable_warning: "{{ bimanual|lower }}"
```

Comment:
Bool or lower?

Reply:
(I mean it would be weird to run the oneliner with bimanual=True but it could happen)

Reply:
Then there is another instance that you need to replace it with lower?

Reply:
Ah yes, thanks! (tested this and both actually worked but I will stick with lower)